cumulus/tasks/pdr-status-check/schemas/input.json

```diff
@@ -23,6 +23,14 @@
     },
     "counter": { "type": "integer" },
     "limit": { "type": "integer" },
-    "isFinished": { "type": "boolean" }
+    "isFinished": { "type": "boolean" },
+    "pdr": {
```

Comment:
Please add a description.

Reply:
added description and made it a required field.